sql/2023/11/ttfb-by-language.sql

```diff
@@ -0,0 +1,83 @@
+# HTTP Archive query to get TTFB information of WordPress sites by language.
```

Comment:
Given the recent change in what this query does, let's update the description (and file name as well?). For example:
```suggestion
# HTTP Archive query to get TTFB information of WordPress sites by whether they are localized or not.
```